### ISSUE TITLE
doesn't keep the columns order when write with "tfrecords"  format from existence Dataframe

### ISSUE THREAD

**Issue by u650080** [tensorflow/ecosystem#117]
Hi ,
I have an existence hdfs file in parquet format. When I write this dataframe  in "tfrecords" format and later  I read  this  new file( in "tfrecords" format)  , the order of columns are changed and it didn't keep same order as parquet format .

**Comment by nareshr8** [comment]
Yeah I also faced the same issue. If you are using the data for training in tensorflow, you can use `tf.feature_column`. But beware of the issue mentioned [here](https://github.com/tensorflow/tensorflow/issues/30453)
